gnomad/resources/grch38/reference_data.py

```diff
@@ -322,6 +322,15 @@ def _import_ensembl_interval(path) -> hl.Table:
     },
 )
 
+# Methylation scores range from 0-12.
```

Comment:
Are these from the same `(https://www.biorxiv.org/content/10.1101/2022.03.20.485034v2.full).` or if not is there a comment we can add indicating where they came from?